public/stylesheets/style.css

```diff
@@ -1,9 +1,71 @@
 @import url('https://fonts.googleapis.com/css?family=Cagliostro&display=swap');
 @import url('https://fonts.googleapis.com/css?family=Lato&display=swap');
 
+
+:root {
+	--selection-background-color: #00B7FF;
+	--selection-color: ghostwhite;
+	--layout-header-text-shadow: ghostwhite;
+	--layout-block-box-shadow: black;
+	--question-border: darkgray;
+	--question-color: darkgray;
+	--categories-li-background-color: antiquewhite;
+	--categories-li-color: #4d4944;
+	--layout-nav-ul-border: black;
+	--table-background: ghostwhite;
+	--layout-nav-ul-background-color: rgba(248, 284, 255, .25);
+	--modal-content-box-shadow: rgba(0, 0, 0, .075);
+	--article-container-background-color: rgba(255, 255, 255, .35);
```

Comment:
Can you add the rules you put in the light @media that are missing here?
Can you also add 2 new variables for the article list background rgba(255, 255, 255, 0.5), rgba(255, 255, 255, 0.85) in light and rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.85) in dark? I will change the JS accordingly